acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -132,8 +195,8 @@ def rpc_blocking_request(broker, targets,
     end
   rescue Timeout::Error
     mutex.synchronize do
-      if !have_all_rpc_responses?
-        raise "Didn't receive all PCP responses when requesting puppet run on #{targets}. Responses received were: #{responses.to_s}"
+      if !have_all_rpc_responses?(targets, responses)
+        raise "Didn't receive all PCP responses when requesting #{pxp_module} #{action} on #{targets}. Responses received were: #{responses.to_s}"
```

Comment:
Yes it was broken. Since it's in the rescue block, it usually never executes, so the mistake was overlooked.